nion/swift/DocumentController.py

```diff
@@ -3012,9 +3012,9 @@ def get_action_name(self, context: Window.ActionContext) -> str:
         return self.action_name
 
 
-class ExportAction(Window.Action):
-    action_id = "file.export"
-    action_name = _("Export...")
+class SingleExportAction(Window.Action):
+    action_id = "export.single_export"
```

Comment:
The old `action_id` should not change. It's unlikely that it is used, but the action id's are meant to be stable over long periods of time. Likewise, I'd like to use consistent identifiers, even though your identifier names are better. That means using `file.batch_export` or maybe `file.export_batch` to make it easily distinguishable in an alphabetized list.

Along those same lines, the classes should be in alphabetical order in the file. This is to make it easier to locate them, which is a problem in the huge file. To that end, I'd keep the original name along with the original identifier (for the reasons above): `ExportAction` or perhaps `ExportSingleAction`.

Reply:
Can I ask the silly question of why do classes need to be alphabetical in a file?  Most modern IDEs (including PyCharm) can list classes/functions in a file in alphabetical order anyway.  If a file has enough classes in it that that list isn't enough surely that is indicative that the file could do with being broken down into smaller chunks rather than trying to use the first few letters of a class name to create pseudo-structure - potentially leading to names that are less descriptive of the actual function and instead are named just to keep them together.

Reply:
They're not really totally alphabetical - only within each major section. It's simply something to make it easier when searching for commands. This file should definitely be broken down into more manageable pieces and now might be a sensible time to do that (just before a release). Or maybe just after. Either way, when a file gets split, it's more difficult to get a history of that file for regressions, so I tend not to do it for that reason.

Reply:
NOTE: the important part of my comment is to not rename the existing command.